python/tank_vendor/shotgun_authentication/login_dialog.py

```diff
@@ -48,24 +48,28 @@ def __init__(self, is_session_renewal, hostname="", login="", fixed_host=False,
         self.ui.site.setText(hostname)
         self.ui.login.setText(login)
 
-        # default focus
-        if self.ui.site.text():
-            if self.ui.login.text():
-                self.ui.password.setFocus()
-            else:
-                self.ui.login.setFocus()
-        else:
-            self.ui.site.setFocus()
-
         # set the logo
         self.ui.logo.setPixmap(QtGui.QPixmap(":/shotgun_authentication/shotgun_logo_light_medium.png"))
 
+        if fixed_host:
+            self._disable_widget(
+                self.ui.site,
+                "You can't edit the host, your core is configured for this site."
```

Comment:
This seems a little technical - maybe it should say "The shotgun site has been predefined and cannot be modified" or something?